lib/web/ui/server.go

```diff
@@ -98,6 +106,15 @@ func MakeServer(clusterName string, server types.Server, logins []string, requir
 		}
 	}
 
+	if server.GetKind() == types.KindGitServer &&
+		server.GetSubKind() == types.SubKindGitHub {
+		if github := server.GetGitHub(); github != nil {
+			uiServer.GitHub = &GitHubServerMetadata{
+				Integration:  github.Integration,
+				Organization: github.Organization,
+			}
+		}
+	}
```

Comment:
it seems like this should be in its own method, something like `MakeGitServer`. If its inside here, it gives the impression that this is a "server type" instead of its own thing.